snmp/postfixdetailed

```diff
@@ -500,10 +507,10 @@ my $data=$received."\n".
 	$rardnf."\n".
 	$rarnfqa."\n".
 	$iuscp."\n".
+	$msefl."\n".
```

Comment:
It's much easier to fix in the main repo otherwise users have to re-deploy the application